lib/rubycritic/analysers/smells/flog.rb

```diff
@@ -16,7 +18,13 @@ def initialize(analysed_modules)
       def run
         @analysed_modules.each do |analysed_module|
           add_smells_to(analysed_module)
+          print green "."
```

Comment:
This should also not be repeated in every "run" method, we'll just need to figure out a smart way not to so ;)